packages/static-build/src/index.ts

```diff
@@ -786,6 +786,8 @@ export const prepareCache: PrepareCache = async ({
     await glob('{.shadow-cljs,node_modules}/**', workPath)
   );
 
+  Object.assign(cacheFiles, await glob('**/node_modules/**', workPath));
```

Comment:
Is there a way to get the glob pattern above "Default cache files" to include this pattern as well? That way we don't need to pass over all the files twice?

Reply:
let me try adjusting the previous one - i don't see why it wouldn't include all the paths